packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -162,10 +162,10 @@ export let PageHeader = React.forwardRef(
     };
 
     /* istanbul ignore next */
-    const handleResizeActionBarColumn = (width) => {
+    const handleResizeActionBarColumn = ({ width }) => {
       /* don't know how to test resize */
       /* istanbul ignore next */
-      setActionBarColumnWidth(width);
+      setActionBarColumnWidth({ width });
```

Comment:
I don't know as much about the PageHeader, but how come you've changed how the `actionBarColumnWidth` is set?

ie:
`setActionBarColumnWidth(width);`
vs
`setActionBarColumnWidth({ width });`

It looks like `width` is already destructured.

Reply:
Fixed, good catch.